joconner/ventura.md

```diff
@@ -0,0 +1,9 @@
+# Ventura, CA
+
+Ventura, CA is a beautiful city on the Pacific coastline.
```

Comment:
Hi John, Nice meeting you virtually... Did not realize that you live in Ventura. Seems to be an awesome place.